python/paddle/fluid/layers/control_flow.py

```diff
@@ -2082,7 +2229,12 @@ def static_input(self, x):
     @signature_safe_contextmanager
     def block(self):
         """
-        The block for user to define operators in RNN.
+        The function is used for user to list the operations executed during
```

Comment:
NIT: "user" -> "users", this is general users here. But the change is not important.

Reply:
Done. I remove "for users". It is redundant.